server/render/index.js

```diff
@@ -78,13 +70,19 @@ export function serverRender( req, res ) {
 		context.i18nLocaleScript = '//widgets.wp.com/languages/calypso/' + context.lang + '.js';
 	}
 
-	if ( config.isEnabled( 'server-side-rendering' ) &&
-		context.store &&
-		context.layout &&
-		! context.user ) {
-		context.initialReduxState = pick( context.store.getState(), 'ui', 'themes' );
-		const key = context.renderCacheKey || JSON.stringify( context.layout );
-		Object.assign( context, render( context.layout, key ) );
+	if ( config.isEnabled( 'server-side-rendering' ) && ! context.user ) {
+		if ( context.layout ) {
+			const key = context.renderCacheKey || JSON.stringify( context.layout );
+			Object.assign( context, render( context.layout, key ) );
+		}
+		if ( context.store ) {
+			const title = getFormattedTitle( context.store.getState() );
+			const metas = getMeta( context.store.getState() );
+			const links = getLink( context.store.getState() );
+			context.head = { title, metas, links };
```

Comment:
We wrap this in `context.store` condition, but the Jade template assumes `head.title` will always exist. Do we need to be safer there too? Or remove the `context.store` check?